third-party/gke/package.json

```
@@ -76,11 +76,12 @@
       "https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-config-connector/v1.113.0/install-bundles/install-bundle-workload-identity/crds.yaml",
       "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-networking-recipes/e0c4725bc03a17f7ed7e9690406387de5287ea2b/gateway-api/config/mesh/crd/stable/tdgrpcroute.yaml",
       "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-networking-recipes/e0c4725bc03a17f7ed7e9690406387de5287ea2b/gateway-api/config/mesh/crd/stable/tdmesh.yaml",
-      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-networking-recipes/e0c4725bc03a17f7ed7e9690406387de5287ea2b/gateway-api/config/mesh/crd/experimental/gcpsessionaffinityfilter.yaml",
-      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-networking-recipes/e0c4725bc03a17f7ed7e9690406387de5287ea2b/gateway-api/config/mesh/crd/experimental/gcpsessionaffinitypolicy.yaml",
-      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-networking-recipes/e0c4725bc03a17f7ed7e9690406387de5287ea2b/gateway-api/config/servicepolicies/crd/standard/gcpbackendpolicies.yaml",
-      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-networking-recipes/e0c4725bc03a17f7ed7e9690406387de5287ea2b/gateway-api/config/servicepolicies/crd/standard/gcpgatewaypolicies.yaml",
-      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-networking-recipes/e0c4725bc03a17f7ed7e9690406387de5287ea2b/gateway-api/config/servicepolicies/crd/standard/healthcheckpolicy.yaml",
+      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-gateway-api/a663dec06e3fa9e07fd252480b1f55043dd1863a/config/crd/networking.gke.io_gcpsessionaffinityfilters.yaml",
+      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-gateway-api/a663dec06e3fa9e07fd252480b1f55043dd1863a/config/crd/networking.gke.io_gcpsessionaffinitypolicies.yaml",
+      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-gateway-api/a663dec06e3fa9e07fd252480b1f55043dd1863a/config/crd/networking.gke.io_gcpbackendpolicies.yaml",
+      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-gateway-api/a663dec06e3fa9e07fd252480b1f55043dd1863a/config/crd/networking.gke.io_gcpgatewaypolicies.yaml",
+      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-gateway-api/a663dec06e3fa9e07fd252480b1f55043dd1863a/config/crd/networking.gke.io_healthcheckpolicies.yaml",
+      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-gateway-api/a663dec06e3fa9e07fd252480b1f55043dd1863a/config/crd/networking.gke.io_gcptrafficdistributionpolicies.yaml",
```

Using the tag would be more readable than the commit sha.

```suggestion
      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-gateway-api/v1.4.0/config/crd/networking.gke.io_gcpsessionaffinityfilters.yaml",
      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-gateway-api/v1.4.0/config/crd/networking.gke.io_gcpsessionaffinitypolicies.yaml",
      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-gateway-api/v1.4.0/config/crd/networking.gke.io_gcpbackendpolicies.yaml",
      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-gateway-api/v1.4.0/config/crd/networking.gke.io_gcpgatewaypolicies.yaml",
      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-gateway-api/v1.4.0/config/crd/networking.gke.io_healthcheckpolicies.yaml",
      "https://raw.githubusercontent.com/GoogleCloudPlatform/gke-gateway-api/v1.4.0/config/crd/networking.gke.io_gcptrafficdistributionpolicies.yaml",
```